malcolm/controllers/countercontroller.py

```diff
@@ -14,9 +14,13 @@ def create_attributes(self):
         yield self.counter
 
     @takes()
-    def reset(self):
+    @returns()
```

Comment:
Shouldn't have to do `@returns()` for no return value, maybe Method needs a default empty return map?